### DISCUSSION TITLE
Option for striped colored Device Roles

### DISCUSSION THREAD

**Discussion by davidh2k** [netbox-community/netbox#11756] [Ideas]
### NetBox version

v3.4.4

### Feature type

Change to existing functionality

### Proposed functionality

Is it possible to add a striped coloring option to the device roles for the Rack View. Similiar to what you see if a Device uses the full depth of a Rack:

![image](https://user-images.githubusercontent.com/6970146/218788987-de69589b-4f78-478a-a0e1-30593c1a6666.png)

I'm not 100% certain if this could lead to potential issues with the device view in the devices section, for readability etc.

### Use case

Besides using a different color shade, this could be another option to differentiate very similiar device types. May it either be PoE vs. non PoE Switches or RJ45 Patchpanels for Switching vs. RJ45 Patchpanels for Phones/DECT Basestations.

**Comment by jsenecal** [comment]
Moving this to a discussion as no implementation details were provided.

**Comment by jeremystretch** [comment]
This seems like a bad idea. We utilize the diagonal striping specifically to denote special statuses beyond device role, such as unavailable rack space. Allowing people to assign these to custom roles would lead to confusion.
